pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -2006,6 +2006,54 @@
         emoji: "App Lab: Make a Whack-E-Moji Game"
         how_to: "App Lab How-to: Make a Choose-Your-Own-Adventure App"
 
+  music_lab:
+    introducing: "Introducing"
+    music_lab: "Music Lab"
```

Comment:
Do these need to be separate because of the icon and styling? I think for localization it's better if these are combined into one string, but I'm not sure if we have a way to allow translators to move the icon. @mgc1194 thoughts? Here's a screenshot of what this should look like 

<img width="345" alt="Screenshot 2024-04-16 at 9 31 59 AM" src="https://github.com/code-dot-org/code-dot-org/assets/46464143/bdc792d4-761a-4a54-aac3-b5f4d49b0a7a">

Reply:
Correct, I was planning on adding the icon with either a separate `span` element, or a `::before` on the Music Lab`p` element. @mgc1194 interested to hear your thoughts!

Reply:
I merged this without hearing back, but still need to add video caption strings when we get them so can update/add a new string if needed.

Reply:
Continuous sentences should always be translated as one string.  If we need to introduce styling or icons, then we should have tokens that make it through the translation process.

Reply:
Thanks @breville! Do you know of an existing example of how to do this with an icon, or would it work similarly to how we add in-line links?

Reply:
Just looking elsewhere in the same file, it looks like it could be done with a markdown image, a `%{}` substitution, or straight-up inline HTML.  I'm guessing our preferences are in that same order, though we'd need to test to make sure it works.

Reply:
Closing the loop here: I wasn't able to make any of the above options work, but I found a good workaround with making `**Music Lab**` strong in the string as it is in the [mock](https://www.figma.com/file/2WMErkWkUqY8DNxPCa5hXB/Music-Lab-Launch?type=design&node-id=2390%3A1392&mode=design&t=oDtxgGvnGWrggT9V-1), and using css to add the icon as a `::before` element ([ref](https://github.com/code-dot-org/code-dot-org/pull/58213/files#r1578271253)):

<img width="257" alt="Screenshot 2024-04-24 at 10 22 03 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/c2aea3b2-5811-4fab-95a9-4271b944b507">